java/src/jmri/jmrit/vsdecoder/Steam1Sound.java

```diff
@@ -977,7 +978,8 @@ public void run() {
                 }
                 _sound.stop();
             } catch (InterruptedException ie) {
-                log.error("execption", ie);
+                // kill thread
+                return;
```

Comment:
Shouldn't there be at least a `log.debug()` statement here, in case it happens unexpected?

Reply:
Interrupting a thread should kill it.  That's what it's for.  Back when people just caught `Exception`, logging helped sort out what had actually happened, but here it seems pretty clear.